PlaybackLab/DownloadToOwnSampleApp/app/src/main/java/com/ooyala/sample/lists/BasicPlaybackListActivity.java

```diff
@@ -42,9 +42,9 @@ public void onCreate(Bundle savedInstanceState) {
 
     selectionMap = new LinkedHashMap<String, PlayerSelectionOption>();
     //Populate the embed map
-    selectionMap.put("Widevine DASH Stream", new PlayerSelectionOption("tyb2g2NjE64BuIcLQyS0wEccq4fr5Y03", OoyalaSkinOPTPlayerActivity.class));
-    selectionMap.put("Widevine DASH Downloader", new PlayerSelectionOption("tyb2g2NjE64BuIcLQyS0wEccq4fr5Y03", OfflineDownloadActivity.class));
-    selectionMap.put("Widevine DASH Offline Player", new PlayerSelectionOption("tyb2g2NjE64BuIcLQyS0wEccq4fr5Y03", OfflineSkinPlayerActivity.class));
+    selectionMap.put("Widevine DASH Stream", new PlayerSelectionOption("BjN213NzE6H-9WPurASHfKFOtEak5x6b", OoyalaSkinOPTPlayerActivity.class));
```

Comment:
I'm assuming these are new embed codes. Why are we changing these (is it related to the title of this PR)?